Wiki.md

```diff
@@ -21,38 +21,41 @@ Here you will find all wiki related information with regards to guidelines for c
+We currently have a couple of guides to help you contribute, via the browser, command line, or desktop application.
 
-- [How To Contribute From Your Browser:](Wiki-Contribute-Online) This is currently the easiest way we have to contribute. A simplified version can be found on [here](https://medium.freecodecamp.com/how-to-land-your-first-open-source-contribution-from-your-browser-in-15-minutes-756d9bbf81ad) also.
-- [How to fork our wiki](Wiki-Contribute-Fork) This will show you how to fork our wiki while being able to keep your fork syncronized.
-- [How To Contribute To The Wiki Locally:](Wiki-Contribute-Local-GUI) This is a guide that will show you how to have a local copy so you can make changes from your computer, it also works for contributing to the main repository.
+- [How to Contribute from Your Browser](Wiki-Contribute-Online): This is currently the easiest way we have to contribute. A simplified version can also be found in the Medium post “[How to land your first open source contribution, from your browser, in 15 minutes](https://medium.freecodecamp.com/how-to-land-your-first-open-source-contribution-from-your-browser-in-15-minutes-756d9bbf81ad).”
+- [How to Fork Our wiki](Wiki-Contribute-Fork): This shows you how to fork our wiki and keep your fork synchronized.
+- [How to Contribute to the Wiki Locally](Wiki-Contribute-Local-GUI): This guide shows you how to make changes to the wiki from a local copy on your computer; it also works for contributing to the main repository.
 - [Pull Request Guidelines](PULL_REQUEST_TEMPLATE)
-- [More on how to make a pull request from a fork](Pull-Request-Contribute)
-- [How to Squash Commits:](git-rebase#squashing-multiple-commits-into-one) We required only one commit per Pull Request unless otherwise specified by a moderator.
+- [More on How to Make a Pull Request from a Fork](Pull-Request-Contribute)
+- [How to Squash Commits](git-rebase#squashing-multiple-commits-into-one): We require only one commit per pull request, unless otherwise specified by a moderator.
 
 ### Labels
 
-Before creating your first pull request or starting your work as an issue moderator it is strongly suggested to get familiar with the [labels guide](Wiki-Labels-Guide). This will provide you with a through understanding on our labeling system so you can proper utilize it and keep a stable workflow.
+Before creating your first pull request or starting your work as an issue moderator, it is strongly suggested that you get familiar with the [labels guide](Wiki-Labels-Guide). This will provide you with a through understanding on our labeling system so you can use it properly and maintain a stable workflow.
 
-For labels on a global levels on FreeCodeCamp's repositories you can follow [this](Select-Issues-for-Contributing-Using-Labels) guide.
+For a list of labels used globally throughout FreeCodeCamp’s repositories, consult the guide “[Select Issues for Contributing Using Labels](Select-Issues-for-Contributing-Using-Labels).”
```

Comment:
Use standard single and double quotes which are available in usual keyboards. 

Certainly, they don't look as good as the other non-standard pretty quote (which are used widely in written publications, online or offline) but just for consistency across the wiki and for future wiki contributions, we should keep the standard quotes.